app/src/main/java/solendsdkfacade/mapper/SolendMethodResultMapper.kt

```diff
@@ -0,0 +1,31 @@
+package solendsdkfacade.mapper
+
+import com.google.gson.Gson
+import org.p2p.wallet.utils.fromJsonReified
+import solendsdkfacade.model.SolendMethodResultError
+import solendsdkfacade.model.SolendMethodResultSuccess
+import timber.log.Timber
+
+class SolendMethodResultMapper(
+    // public due to inline
+    val gson: Gson
+) {
+
+    inline fun <reified SuccessType : SolendMethodResultSuccess> fromSdk(sdkResult: String): SolendResult<SuccessType> {
```

Comment:
I think there is no profit from inlining this function.
That's ok for us that we will call a reference to this function and make the operation here.

Reply:
well, it's for the reified type here to remove big `SolendMethodResultSuccess::class.java` in the parameter